app/components/RoadmapComponent/index.tsx

```diff
@@ -9,7 +9,7 @@ export default function RoadmapComponent() {
           <li>Recipe Upload</li>
           <li>Recipe Edit</li>
           <li>Credentials Request</li>
-          <li>Recipe Search</li>
+          <>Recipe Search</>
```

Comment:
```suggestion
          <li>Recipe Search</li>
```
I think you meant to have another <li> item.